src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      class="bar"
+      data-qa="big"
+    >
+
+      <img
+        src="images/Search.svg"
+        alt="imageBig"
+        class="image_type big_image"
+      >
```

Comment:
```suggestion
      <img
        src="images/Search.svg"
        alt="imageBig"
        class="bar__image"
      >
```

Reply:
use BEM naming of classes(fix everywehere please)
https://en.bem.info/methodology/
also you don't need to set 'big_image' here